src/Twilio.Api/TransportException.cs

```diff
@@ -5,11 +5,11 @@
 
 namespace Twilio
 {
-    public class TransportException<T> : Exception

```

Comment:
We catch `TransportException<Twilio.Message>` from within our code, so you would need to update that also.  I'm not sure if we need the type information though. Checkout our usages